pkg/downloader/ipfs/downloader_test.go

```diff
@@ -46,25 +27,18 @@ func randomText(t *testing.T) []byte {
 
 func TestIPFSDownload(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-	cm := system.NewCleanupManager()
-	defer cm.Cleanup(ctx)
 	defer cancel()
 
-	server, err := ipfs.NewNodeWithConfig(ctx, cm, types.IpfsConfig{PrivateInternal: true})
+	t.Skip("Enable test only when IPFS is running")
```

Comment:
maybe just remove this test as from the looks of things it's always skipped? Or do we have plans to unskip later (issue number)?

Reply:
good catch. Fixing in a follow up PR